test/suppose-process.test.js

```diff
@@ -143,7 +118,7 @@ describe('process', function()
     })
   })
 
-  it('should call a method as response', function(done)
+  it_posixOnly('should call a method as response', function(done)
```

Comment:
Adding out own would be pointless, since we've already done that in `helpers/cli-prompt.js`. This test any any similar that we add for Windows is different in that it tests with a platform prompt and not a JavaScript one.